rmgpy/molecule/kekulize.pyx

```diff
@@ -150,52 +150,52 @@ cdef class AromaticRing(object):
     DO NOT use outside of this module. This class does not do any aromaticity perception.
     """
     cdef public list atoms, resolved, unresolved
-    cdef set endoBonds, exoBonds
-    cdef public int endoDOF, exoDOF
+    cdef set endo_bonds, exo_bonds
+    cdef public int endo_dof, exo_dof
 
-    def __init__(self, atoms=None, endoBonds=None, exoBonds=None, endoDOF=-1, exoDOF=-1):
+    def __init__(self, atoms=None, endo_bonds=None, exo_bonds=None, endo_dof=-1, exo_dof=-1):
         self.atoms = atoms
-        self.endoBonds = endoBonds
-        self.exoBonds = exoBonds
-        self.endoDOF = endoDOF
-        self.exoDOF = exoDOF
+        self.endo_bonds = endo_bonds
+        self.exo_bonds = exo_bonds
+        self.endo_dof = endo_dof
+        self.exo_dof = exo_dof
         self.resolved = []
         self.unresolved = []
 
     cpdef update(self):
         """
         Update the degree of freedom information for this aromatic ring.
 
-        `endoDOF` refers to the number of bonds in the ring without fixed bond orders.
-        `exoDOF`  refers to the number of bonds outside the ring without fixed bond orders.
+        `endo_dof` refers to the number of bonds in the ring without fixed bond orders.
+        `exo_dof`  refers to the number of bonds outside the ring without fixed bond orders.
         """
-        cdef int endoDOF, exoDOF
+        cdef int endo_dof, exo_dof
         cdef Bond bond
 
-        endoDOF = 0
-        for bond in self.endoBonds:
+        endo_dof = 0
+        for bond in self.endo_bonds:
             if bond.isBenzene():
                 # Add one dof for each aromatic bond
-                endoDOF += 1
-        exoDOF = 0
-        for bond in self.exoBonds:
+                endo_dof += 1
+        exo_dof = 0
+        for bond in self.exo_bonds:
             if bond.isBenzene():
                 # Add one dof for each aromatic bond
-                exoDOF += 1
-        self.endoDOF = endoDOF
-        self.exoDOF = exoDOF
+                exo_dof += 1
+        self.endo_dof = endo_dof
+        self.exo_dof = exo_dof
 
-        self.processBonds()
+        self.process_bonds()
 
-    cpdef tuple processBonds(self):
+    cpdef tuple process_bonds(self):
```

Comment:
This particular name change is non-backwards compatible (I think all the other ones are fine, I consider the cdef as pretty much internal since they can't be called directly from python).  Is it conceivable that a user could be using it currently?

Reply:
I don't expect anyone to be using this directly. The only thing which should be used externally from this module should be the main `kekulize` function.

I think the only reason I made these cpdef was so that I could test them.